ConsoleApp6/ColoredGarland.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ConsoleApp6
+{
+    public class ColoredGarland : OrdinaryGarland<ColoredLamp>
```

Comment:
this action shows that the OrdinaryGarland name is not the best way to name class in this case. Maybe just Garland would be better?